src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,51 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
-  this.data.tracks.forEach(function(track, index) {
-    var ui = this.container.find('#genes #track-' + index);
-    if (!ui.length) {
-      this.addTrack_();
-    }
-    this.selectGenes_[index].val(track.fileName).trigger('change');
-  }, this);
+
+  // Load binding list and update panel tracks.
+  var params = {
+    type: 'list-binding'
+  };
+  $.get(genotet.data.serverURL, params, function(data) {
```

Comment:
I'm not saying that passing messages between loader/panel is not right. It's just that the logics between signaling could be revised and made cleaner.

For example, if panel has a uniform refresh entry, that this entry can be the only place that signals any loader request.